debezium-testing/debezium-testing-system/README.md

```diff
@@ -20,14 +20,20 @@ mvn org.apache.maven.plugins:maven-dependency-plugin:2.8:get \
     -Dmaven.repo.local=${MAVEN_REPO}
 ```
 
-Add Oracle drivers for oracle connector
+If the tests are run also against Oracle connector, add Oracle drivers for oracle connector
+
 ``` bash
 ORACLE_ARTIFACT_VERSION=$(mvn -q -DforceStdout help:evaluate -Dexpression=version.oracle.driver)
 ORACLE_INSTANTCLIENT_ARTIFACT_VERSION=$(mvn -q -DforceStdout help:evaluate -Dexpression=version.oracle.instantclient)
 ORACLE_ARTIFACT_DIR="${PWD}/oracle-libs/${ORACLE_ARTIFACT_VERSION}.0"
 
 mkdir -p ${ORACLE_ARTIFACT_DIR}
 cd ${ORACLE_ARTIFACT_DIR}
+```
+
+Download [Oracle Instant Client package](https://www.oracle.com/database/technologies/instant-client/downloads.html), unpack it and place `ojdbc8.jar` and `xstreams.jar` to `${ORACLE_ARTIFACT_DIR}` folder.
```

Comment:
I don't think we need to mention `ojdbc8` here, as that's automatically pulled from maven central now in the upstream.  The only required install bit would be `xstreams`.

Reply:
Good point to mention the version @Naros . However, IMHO `ojbc8` needs to be downloaded. In general at this point (possibly on a fresh installed machine), we have only product maven repo in case of testing productized bits, which doesn't contain Oracle bits.